docs/cep/source/source-types/grpc.md

```diff
@@ -2,96 +2,78 @@
+| keystore.file                | The file path of keystore. If this is provided then client authentication is enabled. | \-            | STRING              | Yes      | No      |
+| keystore.password            | The password of keystore | \-            | STRING              | Yes      | No      |
+| keystore.algorithm           | The encryption algorithm to be used for client authentication.   | \-            | STRING              | Yes      | No      |
+| enable.ssl | To enable ssl. If set to true and keystore.file is not given then it will be set to default carbon jks by default.              | FALSE         | BOOL                | Yes      | No      |
+| mutual.auth.enabled          | To enable mutual authentication. If set to true and keystore.file or truststore.file is not given, then it will be set to default carbon jks by default. | FALSE         | BOOL                | Yes      | No      |
+| threadpool.size              | Sets the maximum size of threadpool dedicated to serve requests at the gRPC server.   | 100           | INT                 | Yes      | No      |
+| threadpool.buffer.size       | Sets the maximum size of threadpool buffer server.               | 100           | INT                 | Yes      | No      |
+
+## System Parameters
+
+| Name                | Description         | Default Value  | Possible Parameters   |
+|---------------------|---------------------|-----------------------|---------------------|
+| keyStoreFile        | Path of the key store file.               | \${carbon.home}/resources/security/gdncarbon.jks        | valid path for a key store file         |
+| keyStorePassword    | The password used with key store file.                 | gdncarbon     | valid password for the key store file   |
+| keyStoreAlgorithm   | The encryption algorithm to be used for client authentication. | SunX509        | \-                 |
+| trustStoreFile      | This is the trust store file with the path.  | \${carbon.home}/resources/security/client-truststore.jks | \-                 |
+| trustStorePassword  | This is the password used with trust store file.               | gdncarbon     | valid password for the trust store file |
+| trustStoreAlgorithm | the encryption algorithm to be used for server authentication. | SunX509        | \-                 |
 
 ## Example 1
 
     CREATE SOURCE BarStream WITH (type='grpc', receiver.url='grpc://localhost:8888/org.gdn.grpc.EventService/consume', map.type='json') (message String);
 
-Here the port is given as 8888. So a grpc server will be started on port
-8888 and the server will expose EventService. This is the default
-service packed with the source. In EventService the consume method is
+Here the port is given as 8888. A gRPC server will be started on port 8888 and the server will expose EventService. This is the default
+service packed with the source. In EventService, the consume method is
```

Comment:
@StoyanTc, any idea how to finish this sentence?

Reply:
This is why I'm checking each of these function topics. Improving the format is nice, but the main point is actually to make sure we actually look at each and every one of these things for stuff like you are finding with these and the email sink/source.

Reply:
Done